client/post-editor/editor-html-toolbar/index.jsx

```diff
@@ -517,7 +517,7 @@ export class EditorHtmlToolbar extends Component {
 					onClick={ this.openMediaModal }
 				>
 					<Gridicon icon="image" />
-					<span data-e2e-insert-type="media">{ translate( 'Media' ) }</span>
+					<span data-e2e-insert-type="media">{ translate( 'Media library' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Media Library', { context: 'Sharing: Service'} )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/external-media-labels)